examples/ldap/openldap.schema

```diff
@@ -0,0 +1,54 @@
+# $OpenLDAP$
+## This work is part of OpenLDAP Software <http://www.openldap.org/>.
+##
+## Copyright 1998-2016 The OpenLDAP Foundation.
+## All rights reserved.
+##
+## Redistribution and use in source and binary forms, with or without
+## modification, are permitted only as authorized by the OpenLDAP
+## Public License.
+##
+## A copy of this license is available in the file LICENSE in the
+## top-level directory of the distribution or, alternatively, at
+## <http://www.OpenLDAP.org/license.html>.
+
+#
+# OpenLDAP Project's directory schema items
+#
+# depends upon:
+#	core.schema
+#	cosine.schema
+#	inetorgperson.schema
+#
```

Comment:
Mentioned that here https://github.com/coreos/dex/pull/1022/commits/60637b51ca6ce7b3a5e6fc07545f80606d6531f0#diff-3936bdc133814c090b1aa375b6fe6717R25

Basically, I found it doesn't install them in predictable places. This is stolen from the current LDAP integration tests.